### PR TITLE
MAUI-3660-OverViewUGContent_MAUI_SfCalendar

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
**Description**

- Added UG Content for OverView

### CODE REVIEW

File: MAUI/Calendar/overview.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
+platform: MAUI
+control: SfCalendar
+documentation: ug
+---
+
+# Overview of .NET MAUI Calendar (SfCalendar)
+
+The Syncfusion .NET MAUI Calendar (SfCalendar) is a lightweight calendar control that allows user to easily select a Single, Multiple and a Range of dates. It offers Month, Year, Decade, and Century view options to quickly navigate to the desired date. To restrict the date selection by using the minimum, maximum, enable past dates, and disabled dates.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/overview.md:12]: Check and remove the lightweight

File: MAUI/Calendar/overview.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+   {% endcolumn}
+   {% column %}
+   ![decadeview-in-maui-calendar](images/overview/MultipleViews/decadeview-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![centuryview-in-maui-calendar](images/overview/MultipleViews/centuryview-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Quick navigation**: You can easily navigate to the different view modes in the SfCalendar.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/overview.md:34]: Different view modes used for quick navigation

File: MAUI/Calendar/overview.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+* **Allow view Navigation**: Allows you to navigate to the different view modes through tap interaction on the cell or header when it is enabled. Restrict users from navigating to differnt calendar views and allows you to select the dates in the year, decade and century view cells when this property is not enabled.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/overview.md:36]: Check and improve the content

Don't use "not enabled", use "disabled"

File: MAUI/Calendar/overview.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   {% endcolumn}
+   {% column %}
+   ![decadeview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/decadeview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![centuryview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/centuryview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Enable Past Dates**: Indicates whether the dates are enabled or disabled before the today date. If this property is disabled, you cannot able to select the dates before the today date and the dates before the today date are considered as disabled dates.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/overview.md:53]: Improve content 

Explain the feature, Don't explain value that holds( Indicates whether the dates)

File: MAUI/Calendar/overview.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+   ![enable-past-date-in-maui-calendar](images/overview/enable-past-date-in-maui-calendar.png)
+
+* **Date selection**: The Selection mode is specified in the calendar property enumeration, allows you to select Single, Multiple, and a Range of dates through interaction. It also supports programmatic selection.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/overview.md:57]: Improve content

Explain the feature, not description The Selection mode is specified in the calendar property enumeration

File: MAUI/Calendar/overview.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+   {% endcolumn}
+   {% column %}
+   ![monthview-multiple-selection-in-maui-calendar](images/overview/DateSelection/monthview-multiple-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   !![monthview-range-selection-in-maui-calendar](images/overview/DateSelection/monthview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Limit the date selection range**: You can select a date from a range of dates between a minimum and maximum number of days by setting the minimum and maximum dates (span of days).
```
Comment (Sasikumar3595) [inline MAUI/Calendar/overview.md:71]: Improve content 

"maximum number of days"

File: MAUI/Calendar/overview.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Checked and removed the lightweight word

File: MAUI/Calendar/overview.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Checked and modified the Quick navigation

File: MAUI/Calendar/overview.md (hunk 3 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Checked and improved the content
Used disabled instead of not enabled

File: MAUI/Calendar/overview.md (hunk 4 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Improved enable past date content

File: MAUI/Calendar/overview.md (hunk 5 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Improved the date selection content and removed the content like description

File: MAUI/Calendar/overview.md (hunk 6 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Improved the Limit the date selection range content and explained about the minimum and maximum number of days

File: MAUI/Calendar/overview.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/overview.md:4]: Learn here about the overview of Syncfusion .NET MAUI Calendar (SfCalendar) control, its basic features, and calendar functionalities.

File: MAUI/Calendar/overview.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+   {% endcolumn}
+   {% column %}
+   ![Decade view in .NET MAUI Calendar.](images/overview/maui-decade-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Century view in .NET MAUI Calendar.](images/overview/maui-century-view.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Quick navigation**: You can easily navigate to the desired date using different calendar views in the SfCalendar.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/overview.md:33]: You can easily navigate to the desired date using different calendar views in the `SfCalendar.`

File: MAUI/Calendar/overview.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+* **Enable and disable built-in view switching**: Restricts or enable navigation to different calendar views through interaction.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/overview.md:35]: Restricts or enables navigation to different calendar views by interaction.

File: MAUI/Calendar/overview.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+   {% column %}
+   ![Weekend days highlighted in .NET MAUI Calendar.](images/overview/maui-weekends-dates.png)
+   {% endcolumn %}
+
+   {% column %}
+   ![Special dates highlighted in .NET MAUI Calendar.](images/overview/maui-special-dates.png)
+   {% endcolumn %}
+   {% endmultiple_column %}
+
+* **Right to left (RTL)**: The Calendar's text direction can be changed by using the Right-to-left property. Enables user to work with RTL languages, such as Hebrew and Arabic.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/overview.md:90]: The Calendars' text direction can be changed by using the `Right-to-left` property. Enables the user to work with RTL languages, such as Hebrew and Arabic.

File: MAUI/Calendar/overview.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+* **Change first day of week**: You can customize the starting day of the week. The default first day of the week is Sunday.
+
+   ![First day of week in .NET MAUI Calendar.](images/overview/maui-first-day-of-week.png)
+
+* **Selectable Day Predicate**: Disable the interaction for calendar view dates and easily prevent the selection of weekends by disabling them.
+
+   ![Selectable day predicate in .NET MAUI Calendar.](images/overview/maui-selectable-day-predicate.png)
+
+* **Highlight weekends and special dates**: Can highlight special dates and weekend dates in month view using different background color and textstyles in MAUI Calendar.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/overview.md:78]: In MAUI Calendar, you can highlight special dates and weekend dates in the month view using different background colors and text styles.

File: MAUI/Calendar/overview.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Overview of .NET MAUI Calendar (SfCalendar)
+
+The Syncfusion .NET MAUI Calendar (SfCalendar) allows user to select a Single date, Multiple dates or a Range of dates. It provides Month, Year, Decade, and Century view options for quickly navigate to the desired date. It supports minimum dates, maximum dates, enable past dates, and disabled dates to restrict the date selection.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/overview.md:12]: Syncfusion .NET MAUI Calendar (SfCalendar) is a calendar control that allows users to select a Single date, Multiple dates, or a Range of dates. It provides various view options such as a Month, Year, Decade, and Century, which allow users to quickly navigate to the desired date. It also supports minimum dates, maximum dates, enable past dates, and disabled dates to restrict the date selection.

File: MAUI/Calendar/overview.md (hunk 7 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Description changed

File: MAUI/Calendar/overview.md (hunk 12 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Changed the overview content

File: MAUI/Calendar/overview.md (hunk 8 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Changed the content

File: MAUI/Calendar/overview.md (hunk 9 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Changed the content

File: MAUI/Calendar/overview.md (hunk 11 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Modified the content

File: MAUI/Calendar/overview.md (hunk 10 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/overview.md]: Content changed

**Review by Nijamudeen-Mohamed — APPROVED** [review]
FIne

**Review by Nijamudeen-Mohamed — APPROVED** [review]

**Pull request merged**